capabilities.json

```diff
@@ -178,6 +220,310 @@
                 }
             }
         },
+        "dualKpiValues": {
+            "displayName": "Dual KPI Values",
+            "properties": {
+                "show": {
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "showKpiValuesTop": {
+                    "displayName": "Show top chart KPI values",
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "topChartDefaultKpiValue": {
+                    "displayName": "Top chart default KPI text",
+                    "type": {
+                        "text": true
+                    }
+                },
+                "showKpiValuesBottom": {
+                    "displayName": "Show bottom chart KPI values",
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "bottomChartDefaultKpiValue": {
+                    "displayName": "Bottom chart default KPI text",
+                    "type": {
+                        "text": true
+                    }
+                }
+            }
+        },
+        "dualKpiTitleFormatting": {
+            "displayName": "Dual KPI Title formatting",
+            "properties": {
+                "fontSizeAutoFormatting": {
+                    "displayName": "Auto text size",
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "fontSize": {
+                    "displayName": "Value text size",
+                    "type": {
+                        "formatting": {
+                            "fontSize": true
+                        }
+                    }
+                },
+                "isBold": {
+                    "displayName": "Bold",
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "isItalic": {
+                    "displayName": "Italic",
+                    "type": {
+                        "bool": true
+                    }
+                },
+                "fontFamily": {
+                    "displayName": "Font family",
+                    "type": {
+                        "enumeration": [
```

Comment:
Are you sure that all of fonts work well in the sandbox mode?

Reply:
all specified fonts works fine